lib/services/android-resources-migration-service.ts

```diff
@@ -22,6 +22,22 @@ export class AndroidResourcesMigrationService implements IAndroidResourcesMigrat
 	public async migrate(appResourcesDir: string): Promise<void> {
 		const originalAppResources = path.join(appResourcesDir, AndroidResourcesMigrationService.ANDROID_DIR);
 		const appResourcesDestination = path.join(appResourcesDir, AndroidResourcesMigrationService.ANDROID_DIR_TEMP);
+		const appResourcesBackup = path.join(appResourcesDir, AndroidResourcesMigrationService.ANDROID_DIR_OLD);
+
+		try {
+			await this.tryMigrate(originalAppResources, appResourcesDestination, appResourcesBackup);
+			this.$logger.out(`Successfully updated your project's application resources '/Android' directory structure.${EOL}The previous version of your Android application resources has been renamed to '/${AndroidResourcesMigrationService.ANDROID_DIR_OLD}'`);
+		} catch (error) {
+			try {
+				this.recover(originalAppResources, appResourcesDestination, appResourcesBackup);
+				this.$logger.out("Failed to update resources. They should be in their initial state.");
+			} catch (error) {
+				this.$logger.out(`Failed to update resources.${EOL} Backup of original content is inside "${appResourcesBackup}".${EOL}If "${originalAppResources} if missing copy from backup folder."`);
```

Comment:
`If "${originalAppResources} iS missing...`

Reply:
Why don't we throw the original error here? Currently the command exits with code 0, as we've caught all errors, which is incorrect.